adding HighAndLow and Multiples katas

ATGE opened the pull request.

Commit:
adding HighAndLow and Multiples katas

carledriss commented on src/main/java/org/fundacionjala/coding/german/MultipleThreeAndFive.java:
```diff
@@ -0,0 +1,30 @@
+    /**
+     *
+     */
+    public MultipleThreeAndFive() {
```
This constructor is not necessary

carledriss commented on src/main/java/org/fundacionjala/coding/german/HighAndLow.java:
```diff
@@ -0,0 +1,28 @@
+     *  @param numbers **this is id of table**
+     * @return <code>String</code>
+     */
+
```
remove this unnecessary break line.
Please update all your methods

Commit:
remove constructor is not necessary in
src/main/java/org/fundacionjala/coding/german/MultipleThreeAndFive.java

and remove this unnecessary break line. in
src/main/java/org/fundacionjala/coding/german/HighAndLow.java

carledriss commented:
LGTM!
